test/cql-pytest/test_describe.py

```diff
@@ -970,6 +972,131 @@ def test_hide_cdc_table(scylla_only, cql, test_keyspace):
         for row in ks_tables:
             assert row.table_name == t_name or row.table_name == cdc_log_name
 
+# Verify that the format of the result of `DESC TABLE` targeting a CDC log table
+# has the expected format.
+@pytest.mark.parametrize("test_keyspace",
+                         [pytest.param("tablets", marks=[pytest.mark.xfail(reason="issue #16317")]), "vnodes"],
+                         indirect=True)
+def test_describe_cdc_log_table_format(scylla_only, cql, test_keyspace):
+    with new_test_table(cql, test_keyspace, "p int PRIMARY KEY", "WITH cdc = {'enabled': true}") as table:
+        log_table = f"{table}_scylla_cdc_log"
+        _, log_table_name = log_table.split(".")
+
+        [row] = cql.execute(f"DESC TABLE {log_table}")
+
+        assert row.keyspace_name == test_keyspace
+        assert row.type == "table"
+        assert row.name == log_table_name
+        
+        # The actual content of `row.create_statement` is tested separately in `test_describe_cdc_log_table_layout`.
```

Comment:
Did you mean `test_describe_cdc_log_table_create_statement`? I cannot find `test_describe_cdc_log_table_layout`.